packages/dashboard/src/utils/notifications/analytics.tsx

```diff
@@ -0,0 +1,75 @@
+import { Stack, Group, Button, Text, Anchor } from "@mantine/core";
+import type { NotificationProps } from "@mantine/notifications";
+
+type NotificationPropsWithId = NotificationProps & { id: string };
+
+export const analyticsNotificationId = "analytics";
```

Comment:
Just curious, what is this id used for?

Reply:
Ah this is so that an id can be assigned to a specific notification, so we can reference that notification later and make changes to it.

Here the notification with id `"analytics"` is updated to show _hey thanks for enabling telemetry_ when user clicks enable, and hidden when user click no.